pwndbg/commands/context.py

```diff
@@ -150,6 +150,61 @@ def contextoutput(section, path, clearing, banner="both", width=None):
                                     banner_top= banner in ["both", "top"],
                                     banner_bottom= banner in ["both", "bottom"])
 
+# Watches
+expressions = set()
+expression_commands = {
+    "eval": gdb.parse_and_eval,
+    "execute": lambda exp: gdb.execute(exp, False, True)
+}
+
+parser = argparse.ArgumentParser()
+parser.description = """
+Adds an expression to be shown on context.
+
+'cmd' controls what command is used to interpret the expression.
+eval: the expression is parsed and evaluated as in the debugged language
+execute: the expression is executed as an gdb command
+"""
+parser.add_argument("cmd", type=str, default="eval", nargs="?",
+                    help="Command to be used with the expression. Values are: eval execute")
+parser.add_argument("expression", type=str, help="The expression to be evaluated and shown in context")
+@pwndbg.commands.ArgparsedCommand(parser, aliases=['ctx-watch', 'cwatch'])
+def contextwatch(expression, cmd=None):
+    expressions.add((expression, expression_commands.get(cmd, gdb.parse_and_eval)))
+
+parser = argparse.ArgumentParser()
+parser.description = """Removes an expression previously added to be watched."""
+parser.add_argument("expression", type=str, help="The expression to be removed from context")
+@pwndbg.commands.ArgparsedCommand(parser, aliases=['ctx-unwatch', 'cunwatch'])
+def contextunwatch(expression):
```

Comment:
can we name it `ctx_unwatch` and make the `ArgparsedCommand` change `_` to `-`?

Same above

Reply:
Just that I get it right: Name the function `ctx_unwatch` and `ArgparsedCommand` does the conversion itself and thus obsolete the first alias, right? Sure, can do. Just thought it best to be similar to `contextoutput`

Reply:
Hmm just checked, it is not automatically renamed. As far as I can see `command_name` in `_ArgparsedCommand` is only filled by aliases and if not set it takes`__name__` of the passed function. 
So I might not get you comment right, as I don't see an easy unhacky way to have the function named `ctx_unwatch` and have it called `ctx-unwatch` in `ArgparsedCommand` =(

Reply:
I was thinking of modifying the ArgparsedCommand and honestly, I haven't  checked if what I wanted is doable (or easily doable). Anyway, this is not anything with high priority so we can leave it as it is.